### DISCUSSION TITLE
Extract url fragment (https://example.com/path#fragment)

### DISCUSSION THREAD

**Discussion by MaxGietl** [tokio-rs/axum#2147] [Q&A] [answered]
Hey, is there any way to extract the URL fragment in a handler? I tried using the `OriginalUri` extractor but it only contains the path itself. I need this to handle an OAuth callback of the form `/path#id_token=<TOKEN>&access_token=<TOKEN>`

**Comment by davidpdrsn** [comment, marked as answer]
Unfortunately not. See https://github.com/hyperium/http/issues/127
